internal/pkg/consensus/processor_test.go

```diff
@@ -1,911 +1,911 @@
-
-		result, err := processor.ApplyMessagesAndPayRewards(ctx, stateTree, th.VMStorage(), []*types.UnsignedMessage{msg1, msg2, msg3}, sender, types.NewBlockHeight(0), nil)
-		require.NoError(t, err)
-
-		assert.Nil(t, result[0].Failure)
-		assert.Error(t, result[1].Failure)
-		assert.False(t, result[0].FailureIsPermanent)
-		assert.Nil(t, result[2].Failure)
-	})
-}
-
-func setupActorsForGasTest(t *testing.T, vms vm.StorageMap, fakeActorCodeCid cid.Cid, senderBalance uint64) ([]address.Address, state.Tree) {
-	ctx := context.TODO()
-
-	mockSigner, _ := types.NewMockSignersAndKeyInfo(1)
-
-	addr1, err := address.NewIDAddress(110)
-	require.NoError(t, err)
-	addr2, err := address.NewIDAddress(111)
-	require.NoError(t, err)
-	addr3, err := address.NewIDAddress(112)
-	require.NoError(t, err)
-
-	addresses := []address.Address{
-		mockSigner.Addresses[0], // addr0
-		addr1,                   // addr1
-		addr2,                   // addr2
-		addr3,                   // minerAddr
-	}
-
-	// act1 message recipient
-	act1 := th.RequireNewFakeActorWithTokens(t, vms, addresses[1], fakeActorCodeCid, types.NewAttoFILFromFIL(100))
-
-	// act2 2nd message recipient
-	act2 := th.RequireNewFakeActorWithTokens(t, vms, addresses[2], fakeActorCodeCid, types.NewAttoFILFromFIL(0))
-
-	// minerActor
-	act3 := th.RequireNewAccountActor(t, types.NewAttoFILFromFIL(0))
-
-	// networkActor
-	act4 := th.RequireNewAccountActor(t, types.NewAttoFILFromFIL(10000000))
-
-	cst := hamt.NewCborStore()
-	cid, st := th.RequireMakeStateTree(t, cst, map[address.Address]*actor.Actor{
-		addresses[1]:                 act1,
-		addresses[2]:                 act2,
-		addresses[3]:                 act3,
-		address.LegacyNetworkAddress: act4,
-	})
-	require.NotNil(t, cid)
-
-	// sender account actor
-	th.RequireInitAccountActor(ctx, t, st, vms, addresses[0], types.NewAttoFILFromFIL(senderBalance))
-
-	return addresses, st
-}
-
-func mustSetup2Actors(t *testing.T, balance1 types.AttoFIL, balance2 types.AttoFIL) (address.Address, *actor.Actor, address.Address, *actor.Actor, state.Tree, vm.StorageMap, types.MockSigner) {
-	ctx := context.TODO()
-	cst := hamt.NewCborStore()
-	vms := th.VMStorage()
-	mockSigner, _ := types.NewMockSignersAndKeyInfo(1)
-
-	_, st := requireMakeStateTree(t, cst, map[address.Address]*actor.Actor{})
-
-	addr1 := mockSigner.Addresses[0]
-	act1, _ := th.RequireInitAccountActor(ctx, t, st, vms, addr1, balance1)
-
-	act2, addr2 := th.RequireNewMinerActor(ctx, t, st, vms, addr1, 10, th.RequireRandomPeerID(t), balance2)
-	return addr1, act1, addr2, act2, st, vms, mockSigner
-}
-
-func mustCreateStorageMiner(ctx context.Context, t *testing.T, st state.Tree, vms vm.StorageMap, minerOwner address.Address) (cid.Cid, *actor.Actor, address.Address) {
-	miner, minerAddr := th.RequireNewMinerActor(ctx, t, st, vms, minerOwner, 1000, th.RequireRandomPeerID(t), types.ZeroAttoFIL)
-	stCid, err := st.Flush(ctx)
-	require.NoError(t, err)
-	return stCid, miner, minerAddr
-}
+// import (
```

Comment:
Would love a TODO or Dragons on us reinstating a lot of these test cases once we wire up the new runtime / get new actors in place.  Anything to tell us when we can expect to resurrect these.